release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -127,6 +127,11 @@ With this release, you can enable bring your own public IPv4 addresses (BYOIP) f
 
 You can deploy {product-title} {product-version} in {gcp-first} in the Dammam, Saudi Arabia (`me-central2`) region and in the Johannesburg, South Africa (`africa-south1`) region. For more information, see xref:../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-regions_installing-gcp-account[Supported GCP regions].
 
+[id="ocp-4-16-installation-nvidia-machine-types_{context}"]
+==== Installation on NVIDIA H100 instance types on {gcp-first}
+
+With this release, you can install {product-title} on {gcp-short} using GPU-enabled NVIDIA H100 machines. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installation-gcp-tested-machine-types_installing-gcp-customizations[Tested instance types for GCP].
```

Comment:
As for "For more information...", suggest to also mention the Google doc on "[Accelerator-optimized machine family](https://cloud.google.com/compute/docs/accelerator-optimized-machines)", which tells useful info for end-users. WDYT? Thanks!

Reply:
Good idea!